Code/Legacy/CryCommon/Cry_Matrix34.h

```diff
@@ -896,7 +682,7 @@ struct Matrix34_tpl
     ILINE void SetRow(int i, const Vec3_tpl<F>& v)  {   assert(i < 3);    F* p = (F*)(&m00);    p[0 + 4 * i] = v.x;   p[1 + 4 * i] = v.y;   p[2 + 4 * i] = v.z;       }
 
     ILINE const Vec3_tpl<F>& GetRow(int i) const    {   assert(i < 3); return *(const Vec3_tpl<F>*)(&m00 + 4 * i);  }
-    ILINE const Vec4_tpl<F>& GetRow4(int i) const   {   assert(i < 3); return *(const Vec4_tpl<F>*)(&m00 + 4 * i);  }
```

Comment:
I see that you removed the template argument of Vec4_tpl, that seems a bit confusing, if the only argument is a float, maybe rename the type to `Vec4_tpl_f`?

Reply:
Since Vec4_tpl is typedef to Vec4, I'll just rename the whole thing to Vec4 😄

Reply:
Sounds good

Reply:
done in 1ab1db8